Branch-SDK/Branch-SDK/BNCContentDiscoveryManager.m

```diff
@@ -309,7 +310,7 @@ - (void)indexContentWithTitle:(NSString *)title
     if ([BNCSystemObserver getOSVersion].integerValue < 9) {
         NSError *error = [NSError errorWithDomain:BNCErrorDomain code:BNCVersionError userInfo:@{ NSLocalizedDescriptionKey: @"Cannot use CoreSpotlight indexing service prior to iOS 9" }];
         if (callback) {
-            callback(nil, error);
+            callback([BNCPreferenceHelper preferenceHelper].userUrl, error);
```

Comment:
I wanted to maintain this callback as nonnull as is the primary use case for all getShortUrl calls, but there's not a handy way to get a long link. This is the next best thing. I didn't invest in the long link messiness because I'm sure this code path is rarely executed, if ever.